### ISSUE TITLE
It's not really unfollowing accounts but there is a workaround I found.

### ISSUE THREAD

**Issue by PrinceSangwan** [mifi/SimpleInstaBot#167]
I have tried all the options to unfollow accounts but all of them fail. It looks like it is unfollowing but the following still remain the same. Unless, I open instagram on my browser and reload my profile page moments after the bot unfollow someone which is when the followers actually change.

**Comment by mifi** [comment]
care to record a screencast of this? I don't understand exactly wwhat's happening

**Comment by PrinceSangwan** [comment]
> care to record a screencast of this? I don't understand exactly wwhat's happening

It's basically not unfollowing anyone I follow.

**Comment by mifi** [comment]
> I open instagram on my browser and reload my profile page moments after the bot unfollow someone which is when the followers actually change.

what does this mean? you mean the bot does actually unfollow?

**Comment by PrinceSangwan** [comment]
> > I open instagram on my browser and reload my profile page moments after the bot unfollow someone which is when the followers actually change.
> 
> what does this mean? you mean the bot does actually unfollow?

No. Quite the opposite actually. It fails to unfollow people. It unfollows someone and moves to another person, but when I go recheck, I am still following that person.

**Comment by mifi** [comment]
Gotcha. possibly related #181
would be nice to see the HTML of the unfollow button and any logs

**Comment by saabtech** [comment]
The new layout unfollow works in latest BOT update.
